kubernetes/linux/setup.sh

```diff
@@ -35,7 +35,7 @@ tar -zxvf telegraf-1.20.3_linux_amd64.tar.gz
 
 mv /opt/telegraf-1.20.3/usr/bin/telegraf /opt/telegraf
 
-chmod 777 /opt/telegraf
+chmod 544 /opt/telegraf
```

Comment:
can you please update this as needed for metrics as well ? :)

Reply:
Yes. I have that in my list of things. I will include that in my PR for feb bug fixes.